### ISSUE TITLE
bad color for solarized color theme...

### ISSUE THREAD

**Issue by valden** [winterTTr/ace-jump-mode#18]
I'm using dark solarized color theme

[submodule "color-theme-solarized"]
        path = color-theme-solarized
        url = git://github.com/sellout/emacs-color-theme-solarized.git

So. when I'm typing C-c SPC and entering 1st letter I'm getting only red letters without original text. It seems you are using solarized background color in your script... 
I see 2 solutions of that problem:
1. Make that color configurable by end user.
2. That color should depends on background one. Use some algorythm to calculate foreground color (f.e. fc = !bc; :) )

**Comment by winterTTr** [comment]
Yes, currently, the foreground color for ace jump candidate is almost a fixed color. I indeed do not give a solution for the dynamic color from context of current theme. Of course, I will try to give a solution for the problem you met, which will dynamically generate the color later.

So if you hurry to fix it right now, I suggest to change `ace-jump-face-background` to make the dimmed word obvious enough.

```
;;; define the face
(defface ace-jump-face-background
  '((t (:foreground "gray40")))
  "Face for background of AceJump motion"
  :group 'ace-jump)
```

the current setting for it is as above.
